WordPress/WordPressUITests/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -141,4 +142,14 @@ class BlockEditorScreen: BaseScreen {
     static func isLoaded() -> Bool {
         return XCUIApplication().navigationBars["Gutenberg Editor Navigation Bar"].buttons["Close"].exists
     }
+    @discardableResult
+    func openBlockPicker() -> BlockEditorScreen {
+        addBlockButton.tap()
+        return BlockEditorScreen()
+    }
+    @discardableResult
+    func closeBlockPicker() -> BlockEditorScreen {
+        offBlockPickerCoordinate.tap()
```

Comment:
The test is consistently hanging for me while instantiating the `BlockEditorScreen`. Instead of defining an `offBlockPickerCoordinate` constant, which is causing it to hang, how about calculating the coordinate directly within this method like this:

```suggestion
        editorCloseButton.coordinate(withNormalizedOffset: CGVector(dx: 0, dy: 0)).tap()
```